lib/auth/init.go

```diff
@@ -470,25 +465,34 @@ func isFirstStart(authServer *AuthServer, cfg InitConfig) (bool, error) {
 // is made to the certificate authority to generate a host certificate and it's written to disk.
 // If a certificate exists on disk, it is read in and returned.
 func initKeys(a *AuthServer, dataDir string, id IdentityID) (*Identity, error) {
-	kp, cp := keysPath(dataDir, id)
+	path := keysPath(dataDir, id)
+
+	keyExists, err := pathExists(path.key)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
 
-	keyExists, err := pathExists(kp)
+	sshCertExists, err := pathExists(path.sshCert)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
 
-	certExists, err := pathExists(cp)
+	tlsCertExists, err := pathExists(path.tlsCert)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
 
-	if !keyExists || !certExists {
+	if !keyExists || !sshCertExists || !tlsCertExists {
```

Comment:
Won't this cause SSH cert and key to be overwritten for existing clusters that do not have TLS cert yet?

Reply:
yes, but that's okay